Tools/CLSGenerator.sh

```diff
@@ -0,0 +1,180 @@
+                choice_successful=true
+                ;;
+            4) # HatEffects
+                item_failed=false
+                copy_file "$sourceDataPath/hateffectinfo/hateffectinfo.lub" "$destinationDataPath/hateffectinfo/hateffectinfo.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/hateffectinfo.lub" "$destinationDataPath/cls/hateffectinfo.lub" || item_failed=true
+                copy_status=$([[ "$item_failed" = true ]] && echo 1 || echo 0)
+                choice_successful=true
+                ;;
+            5) # NPC/Mob and Pets
+                item_failed=false
+                copy_file "$sourceDataPath/cls/jobname.lub" "$destinationDataPath/cls/jobname.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/jobname_f.lub" "$destinationDataPath/cls/jobname_f.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/npcidentity.lub" "$destinationDataPath/cls/npcidentity.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/petinfo.lub" "$destinationDataPath/cls/petinfo.lub" || item_failed=true
+                copy_status=$([[ "$item_failed" = true ]] && echo 1 || echo 0)
+                choice_successful=true
+                ;;
+            6) # Lapine Boxes
+                item_failed=false
+                copy_file "$sourceDataPath/datainfo/lapineddukddakbox.lub" "$destinationDataPath/datainfo/lapineddukddakbox.lub" || item_failed=true
+                copy_file "$sourceDataPath/datainfo/LapineUpgradeBox.lub" "$destinationDataPath/datainfo/LapineUpgradeBox.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/lapineddukddakbox.lub" "$destinationDataPath/cls/lapineddukddakbox.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/lapineupgradebox.lub" "$destinationDataPath/cls/lapineupgradebox.lub" || item_failed=true
+                copy_status=$([[ "$item_failed" = true ]] && echo 1 || echo 0)
+                choice_successful=true
+                ;;
+            7) # Signboards
+                item_failed=false
+                copy_file "$sourceDataPath/cls/signboardlist.lub" "$destinationDataPath/cls/signboardlist.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/signboardlist_f.lub" "$destinationDataPath/cls/signboardlist_f.lub" || item_failed=true
+                copy_status=$([[ "$item_failed" = true ]] && echo 1 || echo 0)
+                choice_successful=true
+                ;;
+            8) # Robes/Costume Garments
+                item_failed=false
+                copy_file "$sourceDataPath/cls/spriterobeid.lub" "$destinationDataPath/cls/spriterobeid.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/spriterobename.lub" "$destinationDataPath/cls/spriterobename.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/spriterobename_f.lub" "$destinationDataPath/cls/spriterobename_f.lub" || item_failed=true
```

Comment:
Remove this.

Reply:
Done on https://github.com/llchrisll/ROenglishRE/pull/100/commits/0e67ea0ed5152e0ed3775913935022d1643dbb67#diff-51a4c473a095a632f039964dc2a80c0e4c5794873ab1cfccfcbdffdf7114bb3bL114